### PR TITLE
#159965477 Send email for verification

### PR DESCRIPTION
**Pull request by Derrickkip**
## What does this PR do?
```
Adds email verification by sending email to the user
```

## Description of Task to be completed?
```
Once the user signs up, an email address is sent to the email address the user 
signed up with

```

## How should this be manually tested?

```
1. Clone this repository
2. Checkout ft-send-email-verification-159965477
3. pip install -r requirements.txt this should also install celery
4. If on a Mac, run `brew install rabbitmq` to install rabbitmq message brocker
5. Run rabbitmq using `brew services start rabbitmq`
6. Run `celery -A authors worker -l info` to start celery
7. Run the server python manage.py runserver
8. navigate to 127.0.0.1:8000/api/users
9. Signup using your email address
10. Check your inbox, there should be a verification email

```
## Any background context you want to provide?

```
When users signup to our application, we want to ensure that the email used is valid. 
Therefore send a token to the email used to register and the user has to click on 
the link in the email to activate their account

```

## What are the relevant pivotal tracker stories?

[#159965477](https://www.pivotaltracker.com/story/show/159965477)

## Screenshots if available to support PR.

screen shot 2018-09-07 at 19 58 47

### CODE REVIEW

**Comment by ghost** [comment]
@Derrickkip Please write your comments according to the teams coding convections.

File: authors/celery.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+app.config_from_object('django.conf:settings', namespace='CELERY')
+
+# app.autodiscover_tasks(lambda: settings.INSTALLED_APPS) # does nothing
+app.autodiscover_tasks()  # also does nothing
+
+print('Registering debug task...')
+
+
+@app.task(bind=True)
+def debug_task(self):
```
Comment (ghost) [inline authors/celery.py]: Please add comments for each method

**Comment by DerKip** [comment]
I suggest that you include the snippet for starting rabbitmq and celery in your PR

**Comment by Derrickkip** [comment]
@DerKip sorry for that updating it right now

**Comment by Derrickkip** [comment]
@DerKip @rethes @Quantum-35 @Mutuba @Teddykavooh Please review this Pr

**Review by Quantum-35 — APPROVED** [review]
LGTM

**Review by ghost — APPROVED** [review]
@Derrickkip I suggest that you include the snippet for starting rabbitmq and celery in the Readme

**Review by WNjihia — APPROVED** [review]

**Pull request merged**